examples/thesis/thesis.md

```diff
@@ -0,0 +1,70 @@
+```config
+lang = "en"
+document_type = "thesis"
+title = "Fancy Title"
+subtitle = "Subtitle (e.g. title in native language)"
+author = "My Name"
+date = "\\today"
+supervisor = "My Supervisor"
+advisor = "My Advisor"
+logo_university = "https://upload.wikimedia.org/wikipedia/commons/thumb/4/48/Markdown-mark.svg/40px-Markdown-mark.svg.png"
```

Comment:
Maybe use: https://cdn.pixabay.com/photo/2017/12/17/14/12/bitcoin-3024279_960_720.jpg
See https://pixabay.com/illustrations/bitcoin-cryptocurrency-virtual-3024279/ for license.

Reply:
This is horribly fantastic :D